earthpy/tests/test_plot.py

```diff
@@ -28,27 +27,27 @@ def test_num_titles():
     should raise an error OR if the title list is a different length than
     the array it should also raise an errors"""
 
-    single_band = im[0]
+    single_band = im_arr[0]
 
     with pytest.raises(ValueError):
         es.plot_bands(arr=single_band,
                       title=["Title1", "Title2"])
     with pytest.raises(ValueError):
-        es.plot_bands(arr=im,
+        es.plot_bands(arr=im_arr,
                       title=["Title1", "Title2", "Title3"])
 
 def test_num_axes():
     """If provided with a 2 band array, plot_bands should return 3 axes by
     default"""
-    fig, ax = es.plot_bands(im)
+    fig, ax = es.plot_bands(im_arr)
     assert len(fig.axes) == 3
 
 
 def test_two_plot_title():
```

Comment:
Using a fixture here could improve clarity, and the `# Get titles` comment seems unnecessary, because of the `.get_title()` method, e.g.,: 

```python
def test_two_plot_title(image_array_3d):
    """Test that the default title is provided for a 2 band array plot"""
    fig, ax = es.plot_bands(image_array_3d)
    ax = fig.axes
    num_plots = image_array_3d.shape[0]
    plot_titles = [ax[i].get_title() for i in range(num_plots)]
    assert plot_titles == ['Band 1', 'Band 2']
```

Reply:
sounds good! i do'nt think i need any comments in this module but wasn't sure if other people who are newer to MPL objects might find the comments helpful. i'd rather remove them all as I think the methods in the obj are pretty darn expressive!!! i removed this! 

as for the fixture for plot elements i'm going to push this and maybe we can chat briefly on slack.

Reply:
This suggestion was to use `image_array_3d` as a fixture, rather than using fixtures for plot elements. So, I think we are on the same page looking at the changes you had made!